src/NLog.AWS.Logger/AWSTarget.cs

```diff
@@ -200,6 +200,19 @@ public string LibraryLogFileName
             set { _config.LibraryLogFileName = value; }
         }
 
+        /// <summary>
+        /// Gets the FlushTimeout property. When performing a flush of the in-memory queue this is the maximum period of time allowed to send the remaining
+        /// messages before it will be aborted. If this is exceeded, incoming log messages will be dropped.
+        /// <para>
+        /// The default is 30 seconds.
+        /// </para>
+        /// </summary>
+        public TimeSpan FlushTimeout
```

Comment:
Maybe call it `FlushTimeoutSeconds` ? (Right now If using FlushTimeout = 30, then it means 30 days).

Reply:
But it's a timespan, not an int.

Reply:
The property-type should ofcourse match the name. Making it easier to configure from `NLog.config`. Like this:
```c#
        public int FlushTimeoutSeconds
        {
            get { return (int)_config.FlushTimeout.TotalSeconds; }
            set { _config.FlushTimeout = TimeSpan.FromSeconds(value); }
        }
```

Reply:
There are other timespans in the config. I was keeping with the already established approach for configuring timespans for consistency.

Reply:
Oh well. Just trying to make things better.

Reply:
Cheers. Need to get someone from Amazon to look at it, otherwise it'll never get merged either way.